.cypress/integration/5_trace_analytics_services.spec.js

```diff
@@ -72,11 +71,6 @@ describe('Testing services table', () => {
 
 describe('Testing service view empty state', () => {
   beforeEach(() => {
-    // exception is thrown on loading EuiDataGrid in cypress only, ignore for now
-    cy.on('uncaught:exception', (err, runnable) => {
```

Comment:
I LOVE that we're removing this.  This is a super-wide ignore nad the test will be far more accurate with uncaught:exceptions bubbling.

Thanks for making this happen!